### PR TITLE
Added more theme templates for login and edit profile pages

### PR DESCRIPTION
**Pull request by ruands**

### CODE REVIEW

File: authentication_service/views.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ThemeMixin:
     sets the appropriate template.
     """
     TEMPLATE_PREFIX = ""
-    theme = None
+    # theme = None
```
Comment (cobusc) [inline authentication_service/views.py]: Can this be removed?

Comment (cobusc) [reply authentication_service/views.py]: Or should it stay?

Comment (ruands) [reply authentication_service/views.py]: Oh, yes 👍

**Comment by cobusc** [comment]
:+1: Just that one comment that needs to be resolved, please.

**Pull request closed without merging**